imtest/imtest.c

```diff
@@ -2880,13 +2890,14 @@ int main(int argc, char **argv)
 
     /* keep this in alphabetical order */
     static const char short_options[] =
-        "?I:P:X:a:cf:hik:l:m:n:o:p:qr:st:u:vw:x:z";
+        "?I:P:X:H:a:cf:hik:l:m:n:o:p:qr:st:u:vw:x:z";
```

Comment:
The comment above says "keep this in alphabetical order", so `H:` should go between `?` and `I:` 😅

Reply:
Done